test/integration-tests/test-server/db-preparation.ts

```diff
@@ -4,7 +4,7 @@ const realityIdHeader = (realityId: number) => ({ headers: { realityId } });
 export const prepareDb = async () => {
     const books = [];
     for (let i = 0; i < titles.length; i++) {
-        books.push({ author: 'chen', title: titles[i], id: i });
+        books.push({ author: 'arik', title: titles[i], testId: i, dataVersion: i + 1 });
```

Comment:
split db preparation files to each int test and init db with the int tests own entities

Reply:
its copying the same code over and over, maybe change the name to basic db preparations? for more special case tests then u will do also a separate initialization?

> split db preparation files to each int test and init db with the int tests own entities

its copying the same code over and over, maybe change the name to basic db preparations? for more special case tests then u will do also a separate initialization?

Reply:
> split db preparation files to each int test and init db with the int tests own entities

its copying the same code over and over, maybe change the name to basic db preparations? for more special case tests then u will do also a separate initialization?

Reply:
ok